flexmeasures/data/services/scheduling.py

```diff
@@ -27,13 +28,37 @@
 from flexmeasures.data.services.utils import job_cache
 
 
+def trigger_fallback_callback(job, connection, type, value, traceback):
```

Comment:
Nice method name. Let's `call_that_fallback_callback`.. :smile: But seriously, I think we can dispense with the `callback` part and just call it `trigger_fallback`. The example functions in the [rq docs](https://python-rq.org/docs/#failure-callbacks) also leave out this noun.

Reply:
hahaha